Using-browsec-for-securing-your-connection-to-fcc.md

```diff
@@ -0,0 +1,63 @@
+# Using Browsec for Securing your Connection to FreeCodeCamp.com
+
+## Why do I need an plugin?
+Lately some of the campers started facing a strange problem, namely content blocking by some Internet Service Providers (ISP), leading to the [FreeCodeCamp.com](http://freecodecamp.com) (FCC) website being broken sometimes.
+
+This came soon to light with some of the issues logged by campers :
+- [\#5999 : Indian ISP's injecting Ads breaks site](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/5999)
+- [\#6122 : Serious Problem using the FCC IDE, console showing error...](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/6122)
+- [\#6381 : Unable to view the Code editor in the browser](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/6381)
+
+## TL;DR : What's Wrong, again?
+
+Well for some campers this is a classic case of man in the middle attack, for instance in the issue \#5999 the ISP some of the campers from India are using is deliberately, injecting Ads into the user's visited websites that's causing the issue.
+
+Sometimes, some ISPs use a similar strategy to cache the content of the websites visited by the consumers, so that they can serve the same to other consumers, this saves them bandwidth costs, while making the websites available faster.
+
+But when this is not done correctly (or is done malaciously as in the first case), this blocks essential content for websites to run seemlessly.
+
+## What's the fix, then?
+
+Simple, we need to anonymize ourselves i.e connect to the FCC website in a secure way, by-passing the ISP, or the firewall caches, whatsoever.
```

Comment:
Isn't it more accurate to say that we need to encrypt our traffic in a way that it cannot be modified during transmission?